### PR TITLE
beanstalk plugin

### PR DESCRIPTION
**Pull request by fatpat**

### CODE REVIEW

File: beanstalk/beanstalk_test.go [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+		_, err = out.Write(b)
+		if err != nil {
+			fmt.Print(err)
+			break
+		}
+		out.Flush()
+	}
+}
+
+func TestMemcachedCollector(t *testing.T) {
```
Comment (vdombrovski) [inline beanstalk/beanstalk_test.go]: TestBeanstalkdCollector

File: cmd/beanstalk/beanstalk.plugin.go [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+	fs := flag.NewFlagSet("", flag.ExitOnError)
+	fs.StringVar(&targets, "targets", "", "Comma separated list of Redis IP:PORT")
+	err := fs.Parse(os.Args[2:])
+	if err != nil {
+		log.Fatalln("ERROR: Memcached plugin: Could not parse args", err)
+	}
+	intervalSeconds := collector.ParseIntervalSeconds(os.Args[1])
+
+	if targets == "" {
+		log.Fatalln("ERROR: Memcached plugin: missing targets")
```
Comment (vdombrovski) [inline cmd/beanstalk/beanstalk.plugin.go]: Beanstalk

File: cmd/beanstalk/beanstalk.plugin.go [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+func main() {
+	if len(os.Args) < 2 {
+		log.Fatalf("argument required")
+	}
+	var targets string
+	fs := flag.NewFlagSet("", flag.ExitOnError)
+	fs.StringVar(&targets, "targets", "", "Comma separated list of Redis IP:PORT")
+	err := fs.Parse(os.Args[2:])
+	if err != nil {
+		log.Fatalln("ERROR: Memcached plugin: Could not parse args", err)
```
Comment (vdombrovski) [inline cmd/beanstalk/beanstalk.plugin.go]: Beanstalk

File: beanstalk/beanstalk_test.go (hunk 1 above)
Comment (fatpat) [reply beanstalk/beanstalk_test.go]: fixed

File: cmd/beanstalk/beanstalk.plugin.go (hunk 3 above)
Comment (fatpat) [reply cmd/beanstalk/beanstalk.plugin.go]: fixed

File: cmd/beanstalk/beanstalk.plugin.go (hunk 2 above)
Comment (fatpat) [reply cmd/beanstalk/beanstalk.plugin.go]: fixed

**Pull request merged**